Arduino/chorus-rf-laptimer/sendSerialHex.h

```diff
@@ -35,6 +35,10 @@ uint8_t HEX_TO_BYTE (uint8_t hi, uint8_t lo) {
     return TO_BYTE(hi)*16+TO_BYTE(lo);
 }
 
+int16_t HEX_TO_UINT16 (uint8_t v4, uint8_t v3, uint8_t v2, uint8_t v1) {

```

Comment:
let's make it using 2 calls to HEX_TO_BYTE and shifting instead of multiplying with numeric constants